{"name": "test_login", "status": "broken", "statusDetails": {"message": "AttributeError: png", "trace": "self = <testCases.test_login.Test_001_Login object at 0x000001E17D324B50>\nsetup = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"4e96dada35b73d143d33262167618ff0\")>\n\n    @pytest.mark.sanity\n    @pytest.mark.regression\n    @allure.severity(allure.severity_level.BLOCKER)\n    \n    def test_login(self,setup):\n        self.logger.info(\"********** Verifying Login test ***********\")\n    \n        self.driver = setup\n        self.driver.get(self.baseURL)\n        # now need to create object of ba\n        # se class (means we call the Login)\n        self.lp = Login(self.driver)\n        self.lp.setusername(self.username)\n        self.lp.setpassword(self.password)\n        self.lp.clicklogin()\n        act_title = self.driver.title\n    \n        if act_title==\"Dashboard / nopCommerce administration13\":\n            assert True\n            self.driver.close()\n            self.logger.info(\"****** Login test is passed ******\")\n    \n    \n        else:\n            # self.driver.save_screenshot(\".\\\\Screenshots\\\\\" + \"test_login.png\")\n>           allure.attach(self.driver.get_screenshot_as_png(),name=\"testloginpage\", attachment_type=AttachmentType.png)\n\ntestCases\\test_login.py:68: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ncls = <enum 'AttachmentType'>, name = 'png'\n\n    def __getattr__(cls, name):\n        \"\"\"Return the enum member matching `name`\n    \n        We use __getattr__ instead of descriptors or inserting into the enum\n        class' __dict__ in order to support `name` and `value` being both\n        properties for enum members (which live in the class' __dict__) and\n        enum members themselves.\n    \n        \"\"\"\n        if _is_dunder(name):\n            raise AttributeError(name)\n        try:\n            return cls._member_map_[name]\n        except KeyError:\n>           raise AttributeError(name) from None\nE           AttributeError: png\n\nC:\\Program Files\\Python39\\lib\\enum.py:352: AttributeError"}, "attachments": [{"name": "log", "source": "761718a0-4261-4243-899e-753e16dd415c-attachment.txt", "type": "text/plain"}], "start": 1609872310311, "stop": 1609872323721, "uuid": "3696036d-0ed9-4d48-827f-e5eda4003588", "historyId": "1d8e0740c42db1e65c856abfafa94fa2", "testCaseId": "6077c220bd96aaae36062a3875e7b04f", "fullName": "test_login.Test_001_Login#test_login", "labels": [{"name": "severity", "value": "blocker"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "sanity"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_001_Login"}, {"name": "host", "value": "DESKTOP-F54TTHH"}, {"name": "thread", "value": "13788-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}